### PR TITLE
Disallow prefixing interfaces with “I”

### PR DESCRIPTION
**Pull request by sindrenm**
According to [the official guidelines][1] from Microsoft, we should “(…) not use ‘I’ as a prefix for interface names”. The default TSLint configuration seems to say otherwise, but personally, I feel like that's a bad idea. Especially when adding types for existing interfaces, this becomes quite tedious since other classes generally don't have that same prefix.

[1]: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names

### CODE REVIEW

**Comment by sindrenm** [comment]
😮

**Comment by timkurvers** [comment]
@sindrenm Dayum son, what happened here? 😮

**Comment by sindrenm** [comment]
I only want an approval, dude! 😆

**Review by timkurvers — APPROVED** [review]
👍

**Comment by sindrenm** [comment]
I also kinda don't have write access. :laughing:

**Comment by sindrenm** [comment]
Yeah, maybe not. :sweat_smile:

**Pull request closed without merging**
